hera_cal/delay_filter.py

```diff
@@ -105,6 +114,12 @@ def load_delay_filter_and_write(infilename, calfile=None, Nbls_per_load=None, sp
         write_cache: bool. If true, create new cache file with precomputed matrices
                            that were not in previously loaded cache files.
         round_up_bllens: bool, if True, round up baseline lengths. Default is False.
+        factorize_flags: bool, optional
+            If True, factorize flags before running delay filter. See vis_clean.factorize_flags.
+        time_thresh : float
```

Comment:
I think that this can be optimized. A straightforward way, to me, is to just vary t_thresh and minimize the fraction of flags in the waterfall. I think I experimented with this a while ago and the fraction appeared to be fairly non-monotonic as a function of t_thresh. We should look into this in the future if we want to fully move to factorized flags (making this decision depends on the severity of the cross talk and the problem where delay filtering on time-dependent flags introduces time-dependent structure in cross talk).